### PR TITLE
some updates on EntityLivingScript, EntityRat, EntityRenderDispatcher…

### PR DESCRIPTION
**Pull request by philhk**
…, EntitySkeletonBoss, EntitySkeletonRifle, EntitySkeletonShotgun, EntitySkeletonSword, EntityZombiePistol

### CODE REVIEW

**Comment by TechPizzaDev** [comment]
LGTM.

File: src/main/java/io/github/ryuu/adventurecraft/overrides/EntityRenderDispatcher.java [hunk 1]
```diff
@@ -67,7 +69,7 @@ private EntityRenderDispatcher() {
         this.o.put(EntityHookshot.class, new RenderHookshot());
         this.o.put(EntityBomb.class, new RenderBomb());
         this.o.put(EntityBat.class, new gv((ko) new ModelBat(), 0.3F));
```
Comment (TechPizzaDev) [inline src/main/java/io/github/ryuu/adventurecraft/overrides/EntityRenderDispatcher.java]: I guess you just forgot this `gv`?

Comment (philhk) [reply src/main/java/io/github/ryuu/adventurecraft/overrides/EntityRenderDispatcher.java]: Oh I did. I did go through it but I missed it. Sometimes my InteliJ doesn't show any errors in a class only after a while :/

Comment (philhk) [reply src/main/java/io/github/ryuu/adventurecraft/overrides/EntityRenderDispatcher.java]: Should be remapped now

**Pull request merged**